src/main/java/database/Storage.java

```diff
@@ -0,0 +1,16 @@
+package database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> assortment = new HashMap<>();
+
+    public static Map<String, Integer> getAssortment() {
```

Comment:
It's great that you're returning a copy of the `assortment` map instead of the map itself. This way you're not exposing your internal data to the outside world. Keep up the good work!